person.rb

```diff
@@ -1,18 +1,8 @@
-require_relative 'nameable'
+class Person
+  attr_accessor :name, :rentals
 
-class Person < Nameable
-  attr_accessor :name, :age
-  attr_reader :id
-
-  def initialize(age, name = 'Unknown', parent_permission: true)
-    super()
-    @id = Random.rand(1..100)
+  def initialize(name)
     @name = name
-    @age = age
-    @parent_permission = parent_permission
-  end
-
-  def correct_name
-    @name
+    @rentals = []
```

Comment:
- [ ] Please create the has-many association here by adding a method showing that a person can have many books. It can look something similar to this:
```
 def add_rental(book, date)
    Rental.new(date, book, self)
  end

```